jsBasics/lesson6/task3.js

```diff
@@ -0,0 +1,19 @@
+
+function checkOrder(available, ordered) {
+
+    if (available < ordered) {
+        console.log("Your order is too large, we don’t have enough goods.");
+        return;
```

Comment:
в Умові було ПОВЕРНУТИ строчку, що тут не зроблено, але не буду знімати бали за консолі. Умови всі правильно зроблені , тому все супер =)  Думаю розуміння раннього повернення у вас є

Reply:
Is this the correct answer?

 function checkOrder(available, ordered) {
    if (available < ordered) {
        return "Your order is too large, we don’t have enough goods.";
    }
    if (ordered === 0) {
        return "Your order is empty.";
    }
    if (available => ordered) {
        return "Your order is accepted."
    }
}

checkOrder(100, 150);
checkOrder(100, 0);
checkOrder(100, 10);

Because if I don't add the third "if" statement and try to get the result in the console.log I get "undefined" when calling function in:
 console.log(checkOrder(100, 10));

<img width="767" alt="undifined" src="https://github.com/user-attachments/assets/85881fd1-ecfc-47eb-a18c-952aab2bc640">

Reply:
Андефайн тому, що останній консоль лог на скріншоті, а має бути return "text";
На скріншоті все вірно окрім останнього консолю.
В коді вище (некстом) не потрібен останній ІФ, просто виконаний return вірно

Reply:
Дякую! Тепер зрозумів і запамʼятаю :)